### PR TITLE
DOCS-4510 Synthetic Monitoring & Continuous Testing Explorer Edits + Search Test Runs Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds Search Test Runs doc page, adds an Explorer section in Synthetics and Continuous Testing doc sets, and renames the CI Results Explorer in several places.

### Motivation

DOCS-4510

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_testing/explorer/_index.md [hunk 1]
```diff
@@ -16,44 +16,69 @@ further_reading:
...
+## Create a search query
+
+Navigate to [**UX Monitoring** > **Explorer**][1] and click on an out-of-the-box search query to start viewing your test batches or test runs and create visualizations.
+
+- View failed tests running in a CI pipeline by filtering on their blocking status and confirming if they are blocking your new releases.
+- Analyze failing test runs with HTTP error status codes to identify API tests with unexpected status codes.
+- Examine test runs that initially failed and passed after a retry.
+- Access test IDs to include in your CI pipeline. 
+
+to add image here
```
Comment (alai97) [inline content/en/continuous_testing/explorer/_index.md:45]: Remember to add an image!

File: content/en/continuous_testing/explorer/_index.md [hunk 2]
```diff
@@ -16,44 +16,69 @@ further_reading:
...
+
+## Explore test runs
+
+The Explorer displays all of your test runs from [Synthetic Monitoring][7] and [Continuous Testing][8]. Every test corresponds to a test run for a particular test subtype, including fast retries. Click on a test in the Explorer to access the test run page.
+
+1. Click on a test run to navigate to the test results or details page. 
+2. Analyze your test run performance, or API and multistep API test performance.
+3. Create a visualization such as a timeseries graph, top list, or table.
+
+to add image here
```
Comment (alai97) [inline content/en/continuous_testing/explorer/_index.md:57]: Remember to add an image!

**Comment by buraizu** [comment]
Thanks, created DOCS-4726 to review

File: content/en/continuous_testing/explorer/search_runs.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+| `Passed`        | The overall status of the test run.                                               |
+| `Run Type`      | The run type of the test run. It can be scheduled, CI, or manually triggered.                                             |
+
+### Timings attributes
+
+**Timings** facets allow you to filter on timing related attributes.
+
+| Facet          | Description                                 |
+|----------------|---------------------------------------------|
+| `DNS`  | The CI provider associated with the batch.  |
```
Comment (peculiarism) [inline content/en/continuous_testing/explorer/search_runs.md:42]: We need to update these descriptions, they slipped through my draft sorry about that! Let's go with the following: 

```
DNS: The time spent resolving the DNS name for an API test run
Download: The time spent downloading the response for an API test run
First Byte: The time spent waiting for the first byte of the response to be received for an API test run
Open: The overall time a websocket remained open for a WebSocket test run
Received: The overall time a websocket connection spent receiving data for a WebSocket test run
TCP: The time spend establishing a TCP connection for an API test run
Total: The total response time for an API test run
```

File: content/en/continuous_testing/explorer/search_runs.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+|------------------|---------------------------------------------------------------------------------------------------------|
+| `Batch ID`        | The batch ID associated with the test run.                                               |
+| <code>Execution&nbsp;Rule</code> | The execution rule associated with the test result of the CI batch: `Blocking`, `Non Blocking`, and `Skipped`. |
+| `Location`       | The location associated with the test result of the batch.                                              |
+| `Passed`        | The overall status of the test run.                                               |
+| `Run Type`      | The run type of the test run. It can be scheduled, CI, or manually triggered.                                             |
+
+### Timings attributes
+
+**Timings** facets allow you to filter on timing related attributes.
```
Comment (peculiarism) [inline content/en/continuous_testing/explorer/search_runs.md:38]:
```suggestion
**Timings** facets allow you to filter on timing related attributes for API test runs.
```

**Review by peculiarism — COMMENTED** [review]
Just a couple of points, let me know what you think!

**Review by peculiarism — APPROVED** [review]

File: content/en/synthetics/_index.md [hunk 5]
```diff
@@ -64,6 +64,10 @@ Analyze performance information about your API tests, multistep API tests, brows
 
 {{< img src="synthetics/test_summary_dashboard.png" alt="Test Summary Dashboard" style="width:100%;">}}
 
+## Use the Synthetic Monitoring & Continuous Testing Explorer
+
+Create [search queries and visualizations][20] for your Synthetic test runs or batches of tests running in CI/CD pipelines.
```
Comment (alai97) [inline content/en/synthetics/_index.md:69]:
```suggestion

{{< img src="continuous_testing/explorer_ci_batches.png" alt="Continuous Testing Explorer" style="width:100%;">}}

Create [search queries and visualizations][20] for your Synthetic test runs or batches of tests running in CI/CD pipelines. 
```

File: content/en/continuous_testing/explorer/_index.md [hunk 6]
```diff
@@ -16,44 +16,69 @@ further_reading:
...
-* Compare test results executed against various devices and browsers to pinpoint cross browser and device issues
-* Keep track of the progress of your tests in CI/CD pipelines
-* Identify flaky tests to fix
+* Compare test runs executed against various devices and browsers to pinpoint cross browser and device issues
+* Examine performance issues with result timing facets and filter runs by failing status codes
+* Try onboarding search queries to get started with searching in the Explorer
+
+## Create a search query
+
+Navigate to [**UX Monitoring** > **Explorer**][1] and click on an out-of-the-box search query to start viewing your test batches or test runs and create visualizations.
```
Comment (alai97) [inline content/en/continuous_testing/explorer/_index.md:38]:
```suggestion
Navigate to [**UX Monitoring** > **Explorer**][1] and click on an out-of-the-box search query to start viewing your test batches or runs and creating visualizations.
```

File: config/_default/menus/menus.en.yaml [hunk 7]
```diff
@@ -2849,58 +2849,63 @@ main:
     parent: synthetics_search
     identifier: synthetics_search_saved_views
     weight: 501
+  - name: Explorer
+    url: synthetics/explorer/
```
Comment (alai97) [inline config/_default/menus/menus.en.yaml:2853]:
```suggestion
    url: synthetics/explorer
```

File: config/_default/menus/menus.en.yaml [hunk 8]
```diff
@@ -2849,58 +2849,63 @@ main:
     parent: synthetics_search
     identifier: synthetics_search_saved_views
     weight: 501
+  - name: Explorer
+    url: synthetics/explorer
```
Comment (alai97) [inline config/_default/menus/menus.en.yaml:2853]:
```suggestion
    url: countinuous_testing/explorer
```

File: config/_default/menus/menus.en.yaml [hunk 9]
```diff
@@ -2849,58 +2849,63 @@ main:
     parent: synthetics_search
     identifier: synthetics_search_saved_views
     weight: 501
+  - name: Explorer
+    url: countinuous_testing/explorer
```
Comment (alai97) [inline config/_default/menus/menus.en.yaml:2853]:
```suggestion
    url: continuous_testing/explorer
```

File: config/_default/menus/menus.en.yaml [hunk 10]
```diff
@@ -2849,58 +2849,63 @@ main:
     parent: synthetics_search
     identifier: synthetics_search_saved_views
     weight: 501
+  - name: Explorer
+    url: continuous_testing/explorer
```
Comment (alai97) [inline config/_default/menus/menus.en.yaml:2853]:
```suggestion
    url: continuous_testing/explorer/
```

File: config/_default/menus/menus.en.yaml [hunk 11]
```diff
@@ -2953,7 +2958,7 @@ main:
     identifier: continuous_testing_testing_tunnel
     weight: 3
   - name: Explorer
-    url: continuous_testing/explorer/
+    url: /continuous_testing/explorer/
```
Comment (alai97) [inline config/_default/menus/menus.en.yaml:2961]:
```suggestion
    url: continuous_testing/explorer/
```

File: content/en/continuous_testing/explorer/_index.md [hunk 12]
```diff
@@ -16,44 +16,69 @@ further_reading:
...
 {{< img src="continuous_testing/explorer_ci_batches.png" alt="Continuous Testing Explorer" style="width:100%;">}}
 
 You can accomplish the following actions:
 
-* Investigate every CI/CD job execution to identify and troubleshoot failing test results
-* Compare test results executed against various devices and browsers to pinpoint cross browser and device issues
-* Keep track of the progress of your tests in CI/CD pipelines
-* Identify flaky tests to fix
+* Compare test runs executed against various devices and browsers to pinpoint cross browser and device issues
+* Examine performance issues with result timing facets and filter runs by failing status codes
```
Comment (rtrieu) [inline content/en/continuous_testing/explorer/_index.md:33]: > * Examine performance issues with _result timing facets_ and filter runs by failing status codes

Once I found this in the UI, I understood what it was, but it's hard to know without context. Maybe a screenshot would help, if you were so inclined!

**Review by rtrieu — CHANGES_REQUESTED** [review]
Left a few suggestions for you!

**Review by rtrieu — APPROVED** [review]
Looking good!

**Pull request merged**